assets/prefabs/food/cooked/Ramen.prefab

```diff
@@ -0,0 +1,24 @@
+{
+    "parent": "engine:iconItem",
+    "Item": {
+        "icon": "Cooking:items#Ramen",
```

Comment:
The `items#` here means to look inside a texture atlas file called `items.png` which is not what you wanted :-) Simply taking it out (which I did real quick) makes it look for `Ramen.png` instead - works now!